frontend/src/pages/home.tsx

```diff
@@ -128,30 +302,58 @@ const Home: FC = () => {
           <span className="font-bold">
             {t('contact-us.cards.call-us.toll-free')}
           </span>{' '}
-          {t('contact-us.cards.call-us.toll-number')}
+          <MuiLink
+          component={Link}
+          href={t('contact-us.cards.call-us.toll-number-href')}
+          >
+            {t('contact-us.cards.call-us.toll-number')}
+          </MuiLink>
         </p>
         <p className="text-lg md:ml-10 md:inline">
           <span className="font-bold">
-            {t('contact-us.cards.call-us.direct')}
+            {t('contact-us.cards.call-us.tty')}
+          </span>{' '}
+          <MuiLink
+          component={Link}
+          href={t('contact-us.cards.call-us.tty-number-href')}
+          >
+          {t('contact-us.cards.call-us.tty-number')}
+          </MuiLink>
+        </p>
+        <p className="mt-3 text-lg">
+          {t('contact-us.cards.call-us.toll-free-description')}
+        </p>
+        <Divider className='my-4'/>
+        <p className="text-lg md:inline">
+          <span className="font-bold">
+            {t('contact-us.cards.call-us.outside')}
           </span>{' '}
-          {t('contact-us.cards.call-us.direct-number')}
+          <MuiLink
+          component={Link}
+          href={t('contact-us.cards.call-us.outside-number-href')}
+          >
+          {t('contact-us.cards.call-us.outside-number')}
+          </MuiLink>
```

Comment:
This should be styled with a `whitespace-nowrap` to prevent the phone number from wrapping in odd places.

Reply:
changed